packages/client/hmi-client/src/utils/facets.ts

```diff
@@ -16,7 +16,7 @@ import { groupBy, mergeWith, isArray } from 'lodash';
 
 // FIXME: this client-side computation of facets from "models" data should be done //////////////////no point in editing//////////////////
 //        at the HMI server
-export const getModelFacets = (models: Model[], conceptFacets: ConceptFacets | null) => {
+export const getModelFacets = async (models: Model[], conceptFacets: ConceptFacets | null) => {
```

Comment:
I don't understand why
- getDatasetFacets, and
- getModelFacets

Are marked as `async`, is there a particular reason here?

Reply:
Generally speaking you'd need async/await pairs if you are dealing with promises, which none were produced here as far as I am able to decipher.

If the idea is that these functions will be replaced with API calls then let's just leave them alone. Otherwise the flow would be a little simpler without the unnecessary async/aait.

Reply:
I think let's leave this alone for now.